src/components/PathwayExercise/ExerciseContent/Assessment/AssessmentContent.js

```diff
@@ -178,47 +200,135 @@ const AssessmentContent = ({
-                      (solution == item.id && classes.correctAnswer)
-                  : answer == item.id && classes.option
-              }
-              onClick={() => !submitDisable && setAnswer(item.id)}
-            >
-              <Stack direction="row" gap={1}>
-                <Typography variant="body1">
-                  {item.value.slice(0, 2)}
-                </Typography>
-                <UnsafeHTML
-                  Container={Typography}
-                  variant="body1"
-                  html={text}
-                />
-              </Stack>
-            </Paper>
-          );
-        })}
+        <Grid container spacing={2}>
+          {Object.values(content.value).map((item, index) => {
+            const text = DOMPurify.sanitize(item.value);
+            const isChecked = answer?.includes(item.id);
+            const isRadioChecked =
+              answer?.length === 1 && answer?.includes(item.id);
+
+            const isValuePresent = solution?.some(
+              (sitem) => sitem.value === item.id
+            );
+            return (
+              <Grid item xs={item.option_type !== "image" ? 12 : 6}>
+                <Paper
+                  elevation={3}
+                  sx={{
+                    height: "auto",
+                    mb: "16px",
+                    cursor: "pointer",
+                    p: "16px",
+                  }}
+                  className={
+                    submit
+                      ? answer?.includes(item.id) && isValuePresent
+                        ? classes.correctAnswer
+                        : answer?.includes(item.id)
+                        ? classes.inCorrectAnswer
+                        : ""
+                      : ""
+                  }
+                  onClick={() => {
+                    if (type === "single") {
+                      if (submit === true) {
+                        return;
+                      } else {
+                        setAnswer([item.id]);
+                      }
+                    } else {
+                      if (submit === true) {
+                        return;
+                      } else {
+                        const updatedAnswer = isChecked
+                          ? answer.filter((id) => id !== item.id)
+                          : [...answer, item.id];
+                        setAnswer();
+                        setAnswer(updatedAnswer);
+                      }
+                    }
+                  }}
+                >
+                  <Stack direction="row" gap={1}>
+                    <FormControlLabel
+                      control={
+                        type === "single" ? (
+                          submit ? (
+                            answer?.includes(item.id) && isValuePresent ? (
+                              <Radio
+                                checked={isRadioChecked}
+                                disabled={submit && true}
```

Comment:
yahan `&& true` kyon upayog ho raha hai? Kyaa `submit` ek truthy non-true maan jaise `1` ho sakte hain?

Reply:
@jschanker Sir,

Here, submit contains a boolean value, which is needed.
If the submit is true, then the option will be disabled.

Reply:
@amansharmma Right, lekin kyon nahi:
```suggestion
                                disabled={submit}
```
matlab `&& true` nahi chahiye.

Agar `submit` `true` hai, toh `submit && true` `true` hoga. Agar `submit` `false` hai, `submit && false` `false` hoga. Phir agar submit Boolean hai, toh `&& true` nahi chahiye kyonki `submit` barabar `submit && true` hoga.  

Main ummeed karta hoon ki meri Hindi samajh mein aayegi. Mujhe abhyas karna hai!

Reply:
Yes, @jschanker Sir,

I got your point now. Thanks for describing.

You are doing well in your Hindi learning:)